### DISCUSSION TITLE
Relation / Expand Query

### DISCUSSION THREAD

**Discussion by CallumVass** [pocketbase/pocketbase#4154] [Q&A] [answered]
Hi,

I have the following 2 tables: `forms` and `sections`

`sections` has a `form_id` single relation to `forms`. I'm trying to query `forms` with `expand` of `sections` but I see this error:
```
DEBUG Failed to enrich list records
└─ map[error:Failed to expand: map[sections:Couldn't find relation field "sections" in collection "forms".]]
```
So my question is, how can I query all sections inside all my forms, for reference this is what I am running:
```
    const resultList = await pb
      .collection("forms")
      .getList<FormsResponse>(1, 50, {
        expand: "sections",
      });
```
Have I got my relation configured wrong?

**Comment by ganigeorgiev** [comment, marked as answer]
If I understand your structure, then you can use [back-relation expand](https://pocketbase.io/docs/working-with-relations/#back-relation-expand):

```js
expand: "sections(form_id)"
```

**Reply by CallumVass** [reply]
Thank you, that worked!
